src/Mod/Assembly/Gui/ViewProviderAssembly.cpp

```diff
@@ -459,6 +475,36 @@ bool ViewProviderAssembly::mouseButtonPressed(int Button,
     return false;
 }
 
+void ViewProviderAssembly::doubleClickedIn3dView()
+{
+    // Double clicking on a joint should start its edition.
```

Comment:
"Edition" == "addition"?

Reply:
Not actually resolved: "edition" is not the word you are looking for here. Try just "edit", or reword to "start editing it" or "enters edit mode" or something.

Reply:
Ah ok